sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestContextManager.java

```diff
@@ -42,12 +45,17 @@ public TestContextManager(Method testMethod, TestMode testMode) {
      * @param enableTestProxy True if the external test proxy is in use.
      * @param recordWithoutRequestBodyClassAnnotation flag indicating if {@code RecordWithoutRequestBody} annotation
      * present on test class.
+     * @param testClassPath the test class path
+     * @param skipExternalizeRecording flag indicating if {@code SkipExternalizeTestRecording} annotation
```

Comment:
In what scenarios do we want the tests to control whether the recording should be externalized or not?